src/app/OperationalSessionSetup.cpp

```diff
@@ -278,6 +279,14 @@ CHIP_ERROR OperationalSessionSetup::EstablishConnection(const ReliableMessagePro
     mCASEClient = mClientPool->Allocate();
     ReturnErrorCodeIf(mCASEClient == nullptr, CHIP_ERROR_NO_MEMORY);
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
```

Comment:
If this is not defined.... shouldn't the useTransportForLargePayload case just error out?